go-controller/pkg/config/config.go

```diff
@@ -238,6 +246,23 @@ type MonitoringConfig struct {
 	IPFIXTargets []HostPort
 }
 
+// IPFIXConfig holds IPFIX-related performance configuration options. It requires that the ipfix-targets
+// value of the [monitoring] section contains at least one endpoint.
+type IPFIXConfig struct {
+	// Sampling is an optional integer in range 1 to 4,294,967,295. It holds the rate at which
+	// packets should be sampled and sent to each target collector. If not specified, defaults to
+	// 400, which means one out of 400 packets, on average, will be sent to each target collector.
+	Sampling uint `gcfg:"sampling"`
+	// CacheActiveTimeout is an optional integer in range 0 to 4,200. It holds the maximum period in
+	// seconds for which an IPFIX flow record is cached and aggregated before being sent. If not
+	// specified, defaults to 60. If 0, caching is disabled.
```

Comment:
same as above, I'd like to clarify if both `CacheActiveTimeout` and `CacheMaxFlows` needs to be 0 to disable caching.

Reply:
Added entry for the next sync meeting